apps/web/components/Charts/v2/renderers/ChartRenderer.tsx

```diff
@@ -39,6 +39,58 @@ type ChartProps = {
   slice: IChart
 }
 
+const getData = (slice: IChart, queryResult: DataItem[]) => {
+  const base = slice.sourceData ? JSON.parse(slice.sourceData) : queryResult
+
+  const values = slice.components.map((component) =>
+    JSON.parse(component.values || '{}'),
+  )
+
+  if (base.length > 0) {
```

Comment:
So if there is anything coming from the base then we skip using the manually provided cms data.

Shouldn't this rather be a check whether there is any manually provided data?